src/nav/style-guide.yml

```diff
@@ -15,7 +15,7 @@ pages:
     pages:
       - title: Capitalization
         path: /docs/style-guide/capitalization/capitalization
-      - title: Product capability and feature capitalization and usage
+      - title: Product, capability and feature capitalization and usage
```

Comment:
```suggestion
      - title: Product, capability, and feature capitalization and usage
```

Reply:
I caught that right after I was all ready to go, so now we'll see what happens. I'm going to merge. Thanks!